build-system/tasks/performance/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "name": "performance",
+  "version": "0.1.0",
+  "description": "Gulp performance",
```

Comment:
Nit: A bit more of a description would be nice!

Reply:
No problem, I just copied the one in the `e2e/package.json` file here. I'm not really sure how this works with nested packages, is that an idiom or does it have an actual effect? What kind of desc would you expect to see?

Reply:
There's no actual effect since this is a private package. I think it's sufficient to make the description something like "Dev-dependencies used by `gulp performance`".

The original sources are here, feel free to modify them as well if you like:

https://github.com/ampproject/amphtml/blob/fe1fb520f20cba637356f9936b284cc60781de1a/build-system/tasks/visual-diff/package.json#L1-L11

https://github.com/ampproject/amphtml/blob/fe1fb520f20cba637356f9936b284cc60781de1a/build-system/tasks/e2e/package.json#L1-L14